lib/magma/project.rb

```diff
@@ -62,9 +62,27 @@ def load_project
         require_files('metrics')
       end
 
+      load_models
       load_model_attributes
     end
 
+    def load_models
+      Magma.instance.db[:models].where(project_name: @project_name.to_s).
+        reject { |model| project_container.const_defined?(model[:model_name].classify) }.
+        each do |model|
+          model_class = Class.new(Magma::Model) do
+            set_schema(
+              model[:project_name].to_sym,
+              model[:model_name].pluralize.to_sym
+            )
+
+            dictionary(model[:dictionary].symbolize_keys) if model[:dictionary]
+          end
+
+          project_container.const_set(model[:model_name].classify, model_class)
```

Comment:
By adding the model constant onto the `project_container`, the anonymous classes become more concrete and respond to methods like `name`.

```ruby
 => 82:           binding.irb
    83:           project_container.const_set(model[:model_name].classify, model_class)
    84:           binding.irb
    85:         end
    86:     end
    87: 

irb(#<Magma::Project:0x0000558bf3ff4fb8>):001:0> model_class
=> #<Class:0x0000558bf402e858>
irb(#<Magma::Project:0x0000558bf3ff4fb8>):002:0> model_class.name
=> nil
```

```ruby
    83:           project_container.const_set(model[:model_name].classify, model_class)
 => 84:           binding.irb
    85:         end
    86:     end
    87: 
    88:     def load_model_attributes
    89:       model_attributes = Magma.instance.db[:attributes].

irb(#<Magma::Project:0x0000558bf3ff4fb8>):001:0> model_class
=> Labors::Aspect
irb(#<Magma::Project:0x0000558bf3ff4fb8>):002:0> model_class.name
=> "Labors::Aspect"
```